README.md

```diff
@@ -22,6 +22,13 @@ MeZO demonstrates comparable performance to fine-tuning with backpropagation acr
   </em>
 </p>
 
+## Installation
+
+
+```bash
+pip install git+https://www.github.com/lebrice/MeZO
```

Comment:
Note: If you're interested in merging this PR, then simply accept this change so people pip-install your repo instead of my fork.
```suggestion
pip install git+https://www.github.com/princeton-nlp/MeZO
```